libmscore/tempotext.cpp

```diff
@@ -190,6 +190,7 @@ void TempoText::textChanged()
                         }
                   }
             }
+      score()->fixTicks();
```

Comment:
I guess this is not needed if the tempo is not actually changed right? So that should move just before score()->setPlaylistDirty();